dartagnan/src/main/java/com/dat3m/dartagnan/program/specification/AssertBasic.java

```diff
@@ -21,6 +21,14 @@ public AssertBasic(Expression e1, COpBin op, Expression e2){
         this.op = op;
     }
 
+    public Expression getE1() {
+        return e1;
+    }
+
+    public Expression getE2() {
+        return e2;
+    }
+
```

Comment:
For consistency with expression, please use `getRHS` and `getLHS`

Reply:
I think we use `getLeft` and `getRight` now (might still be `getLHS/RHS` on your branch though).
On a side note: there is a similar naming problem for assertions in the recent #634 PR.

Reply:
Okay thanks, renamed to `getLeft`/`getRight` now, as well for the `AssertCompiositeAnd `and `AssertCompositeOr `of the above mentioned PR.